### DISCUSSION TITLE
AWS Cloud Personal: Stack Generation: S3-URL not generated

### DISCUSSION THREAD

**Discussion by Jean28518** [mozilla/hubs#5764] [Hubs Cloud Questions]
Hi guys,

I just want to set up a aws cloud personal instance and at the step 1 "Create stack" the Amazon S3 URL isn't filled automatically.

![2022-10-27_18-25](https://user-images.githubusercontent.com/39700889/198346127-872309f1-39e5-4ff4-9207-1c0aedb932ad.png)

I followed the official YouTube Tutorials.
Here is my video how I did the steps: (it was to big for github): https://cloud.server-jean.de/s/AwieTd84pYpfCcH

Do you know where my problem is? Did I do anything wrong? The AWS account is brand new. 

Thanks!
Jean

**Comment by pattersonbl2** [comment]
It doesn't look like you did anything wrong. This looks to be an AWS issue. We will have to reach out to them to see what the issue is.

**Reply by Jean28518** [reply]
Thank you very much!
